media/Config/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <!--
+            Use a content security policy to only allow loading images from https or from our extension directory,
+            and only allow scripts that have a specific nonce.
+        -->
+        <meta http-equiv="Content-Security-Policy"
+        content="img-src https: data:;
+        style-src 'unsafe-inline' ${webview.cspSource};
+        script-src 'nonce-${nonce}';">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="${stylesResetUri}" rel="stylesheet">
+        <link href="${stylesMainUri}" rel="stylesheet">
+    </head>
```

Comment:
With htmlhint of #113, I got these errors (need some time to land these)
Can you please fix them? or if you think these are not necessary, please give your opinion
```
      L16 |    </head>
               ^ <title> must be present in <head> tag. (title-require)
      L42 |                            <input id="useBtn"type="checkbox">
                                       ^ Special characters must be escaped : [ < ]. (spec-char-escape)
      L42 |...       <input id="useBtn"type="checkbox">
                                                      ^ Special characters must be escaped : [ > ]. (spec-char-escape)

Scanned 2 files, found 3 errors in 1 files (15 ms)
```

Reply:
i change html and chekc for vscode extension htmlhint.
L42 problem is from not spacing id and type.
L16 problem is from there is no title.
so i add title in head tag and make space between id and type in L42